domains/POAS/events/uid-requested/examples/example.json

```diff
@@ -0,0 +1,10 @@
+{
+    "lpaID": "bc3d93be-1766-4a90-81d3-01d92abc5aa3",
+    "donorSessionID": "dlVON3lWWjBjamNm",
+    "type": "hw",
+    "donor": {
+        "name": "Jack Rubik",
+        "dob": "1938-03-18",
+        "postcode": "N184EQ"
```

Comment:
Not for this PR but do we need to discuss whether we bring this in line with `application-updated` where we are sending the full address?